ovs/dal/tests/test_alba.py

```diff
@@ -59,7 +60,7 @@ def test_asd_statistics(self):
             'alba_nsm_clusters': [(1, 1)],  # (<abackend_id>, <amount_of_nsm_clusters>)
             'alba_nodes': [1],
             'alba_disks': [(1, 1)],  # (<adisk_id>, <anode_id>)
-            'alba_osds': [(1, 1, 1)]  # (<osd_id>, <adisk_id>, <abackend_id>)
+            'alba_osds': [(1, 1, 1, 1, '1')]  # (<osd_id>, <adisk_id>, <abackend_id>, <anode_id> <slot_id>)
```

Comment:
Why not use integer `1` for all IDs? And if you want it to be a string, convert to string in the helper function